routes/userRoutes.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const User = mongoose.model('user_collection');
```

Comment:
der name der collection in der datanbank ist nur `user` und nicht `user_collection`